react/Header/UserAccount/UserAccount.less

```diff
@@ -107,12 +107,12 @@
   text-overflow: ellipsis;
   display: inline-block;
   white-space: nowrap;
-  max-width: 90px; // ~ 10 characters
+  max-width: 120px; // ~ 12 characters
```

Comment:
These changes don't quite match what we had in #621. Also there are a bunch of other changes in this file in #621. Any reason why we shouldn't revert them all?

Reply:
Yeap, that values are different because the bell is now gone, we can afford to allow a larger space for the names.